src/wp-admin/includes/theme-install.php

```diff
@@ -269,3 +269,59 @@ function install_theme_information() {
 	iframe_footer();
 	exit;
 }
+
+/**
+ * Outputs JS template and conditionals for checking theme compatibility with WordPress and PHP.
+ *
+ * @since 6.4.0
+ *
+ * @return string
```

Comment:
```suggestion
 * Generates JS template and conditionals for checking theme compatibility with WordPress and PHP.
 *
 * @since 6.4.0
 *
 * @return string The JS template.
```
Minor adjustments 🙂